src/components/renderer/form-button.vue

```diff
@@ -1,49 +1,100 @@
 <template>
-  <div class="form-group"  style="overflow-x: hidden">
-    <button v-b-tooltip="options" @click="click" :class="classList" :name="name" :aria-label="$attrs['aria-label']" :tabindex="$attrs['tabindex']">
+  <div class="form-group" style="overflow-x: hidden">
+    <button
+      v-b-tooltip="tooltipOptions"
+      :class="classList"
+      :name="name"
+      :aria-label="ariaLabel"
+      :tabindex="tabIndex"
+      @click="click"
+    >
       {{ label }}
     </button>
   </div>
 </template>
 
 <script>
-import Mustache from 'mustache';
-import { mapActions, mapState } from "vuex";
-import { getValidPath } from '@/mixins';
-
+import Mustache from "mustache";
+import { getValidPath } from "@/mixins";
 
 export default {
   mixins: [getValidPath],
-  props: ['variant', 'label', 'event', 'eventData', 'name', 'fieldValue', 'value', 'tooltip', 'transientData'],
+  props: {
+    variant: {
+      type: String,
+      default: "primary"
+    },
+    label: {
+      type: String,
+      default: "New Submit"
+    },
+    event: {
+      type: String,
+      default: "submit"
+    },
+    eventData: {
+      type: Object,
+      default: null
+    },
+    name: {
+      type: String,
+      default: null
+    },
+    fieldValue: {
+      type: String,
+      default: "1"
+    },
+    value: {
+      type: String,
+      default: "1"
```

Comment:
I don't think we want the default value and fieldValue to be `1` right?

Reply:
right props types and defaults was removed because has causing some compatibilitie errors